DangoPlop/Assets/2DLaserPack/Prefabs/SpriteBasedLasers/BlueSpriteLaser.prefab

```diff
@@ -1,1299 +0,0 @@
-%YAML 1.1
```

Comment:
Don't delete stuff from the package. Just move it to where we need it in our sprites/materials/etc....